clients/go/datacatalog/config.go

```diff
@@ -0,0 +1,37 @@
+package datacatalog
+
+import (
+	"context"
+	"github.com/flyteorg/flytestdlib/config"
+
+	"github.com/flyteorg/flytestdlib/logger"
+)
+
+//go:generate pflags Config --default-var defaultConfig
+
+const ConfigSectionKey = "data-catalog"
+
+var (
+	defaultConfig = &Config{}
+
+	//TODO: Add this section when everybody has migrated
+	//configSection = config.MustRegisterSection(ConfigSectionKey, defaultConfig)
+)
+
+type Config struct {
+	Endpoint              config.URL      `json:"endpoint" pflag:",For admin types, specify where the uri of the service is located."`
+	UseInsecureConnection bool            `json:"insecure" pflag:",Use insecure connection."`
+	MaxBackoffDelay       config.Duration `json:"maxBackoffDelay" pflag:",Max delay for grpc backoff"`
+	PerRetryTimeout       config.Duration `json:"perRetryTimeout" pflag:",gRPC per retry timeout"`
+	MaxRetries            int             `json:"maxRetries" pflag:",Max number of gRPC retries"`
+}
```

Comment:
This config seems to be duplicated here https://github.com/flyteorg/flyteidl/pull/138/files#diff-5d0365f54344929e62dd15277cf7443511432146a43612d8c9471f44a884d7aaR13

Reply:
Is this supposed to be datacatalog only config .